### ISSUE TITLE
Force a space between JSDoc description and properties

### ISSUE THREAD

**Issue by gajus** [eslint/eslint#3839]
Force:

```js
/**
 * Handle special keyboard keys in query input box.
 * @param {SyntheticKeyboardEvent} event
 * @returns {undefined}
 */
```

to be:

```js
/**
 * Handle special keyboard keys in query input box.
 *
 * @param {SyntheticKeyboardEvent} event
 * @returns {undefined}
 */
```

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
@gajus since you're writing a plugin for JSDoc related rules, do you still want to propose this for core?

**Comment by gyandeeps** [comment]
@gajus Can u plz answer the above question?

**Comment by gajus** [comment]
No, I don't. It is covered by https://github.com/gajus/eslint-plugin-jsdoc.
